jbehave-support-core/src/test/groovy/org/jbehavesupport/test/sample/Ssh.story

```diff
@@ -21,3 +14,40 @@ Then the following data are present in [TEST] log:
 | header     | verifier    |
 | ^some.*$   | REGEX_MATCH |
 | unexpected |             |
+
+Scenario: Test log data from specific start and end timestamp
+Given current time is set as a log start timestamp
+!-- Fill you logs (via calling some app etc)
+Given current time is set as a log end timestamp
+Then the following data are present in [TEST] log:
+| header |
+| also   |
+| sharp  |
+Then the following data are present in [TEST] log:
+| header |
+| also   |
+| sharp  |
+
+Scenario: Test log data from specific start and end timestamp saved as into context
+Given log timestamp is saved as [START_1]
+!-- Fill you logs (via calling some app etc)
+Given log timestamp is saved as [END_1]
+Given log timestamp is saved as [START_2]
+!-- Fill you logs again (via calling some app etc)
+Given log timestamp is saved as [END_2]
+
+Given log read start timestamp is set to saved value [{CP:START_1}]
+Given log read end timestamp is set to saved value [{CP:END_1}]
+!-- Read logs from first app
+Then the following data are present in [TEST] log:
+| header |
+| also   |
+| sharp  |
```

Comment:
These tests seems not to be proving presented features. Could you please rework it to more robust state?

Reply:
Ssh story was updated to better mirror presented features.  (It is not testing anything new).
New groovy test was added to test cache feature which it the presented feature. 
Are you OK with this solution?

I agree that we need more robust test of ssh steps, but to do so, we would have to develop some logging mechanism for our web (and/or rest) services  and create "real" logging mechanism in test app. Or something else but this would be most real like.

I would suggest creating enhancement issue for this and do not solve it there as it ssh feature is present in our project before this pull request.